### ISSUE TITLE
Session time out when running geckodriver as SYSTEM user

### ISSUE THREAD

**Issue by ChristianDiedrich** [mozilla/geckodriver#442]
When trying to run geckodriver as the SYSTEM user, the request to create a new session runs into a time out.

In our company we run tests via CruiseControl triggered builds. The CruiseControl.NET server runs as a windows service. With Firefox v47 and below, this was working as expected. But since we tried to switch to Firefox v50 and geckodriver v0.13, all tests fail already at the start when trying to create a new FirefoxDriver instance:

`
            SetUp : OpenQA.Selenium.WebDriverException : The HTTP request to the remote WebDriver server for URL http://localhost:61534/session timed out after 180 seconds.
           ----> System.Net.WebException : Die Anfrage wurde abgebrochen: Timeout für Vorgang überschritten.
            bei OpenQA.Selenium.Remote.HttpCommandExecutor.CreateResponse(WebRequest request)
            bei OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)
            bei OpenQA.Selenium.Remote.DriverServiceCommandExecutor.Execute(Command commandToExecute)
            bei OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary2 parameters)
            bei OpenQA.Selenium.Remote.RemoteWebDriver.StartSession(ICapabilities desiredCapabilities)
            bei OpenQA.Selenium.Remote.RemoteWebDriver..ctor(ICommandExecutor commandExecutor, ICapabilities desiredCapabilities)
`

## Firefox Version
50.1.0

## Platform
Windows

**Comment by andreastt** [comment]
I’m sorry, but it’susually not a good idea to report bugs on old software. When making an issue report, you as an issue-filer should make an effort to not waste the author’s time by making sure you’re reporting a genuine bug.

Your bug report is lacking both the trace-level Gecko log the issue template requested of you, that you test on the latest Firefox, and reproducible steps. Because it’s simply impossible for me to tell what’s wrong with the information available I’m going to close the issue. I’m happy to reopen it once sufficient reproducible information has been added.

**Comment by ChristianDiedrich** [comment]
@andreastt 
I updated the initial report with all the information I could collect. Sorry for not providing all vital information in the first place.

I could not gather the output of the geckodriver.exe window, since this does not exist when being executed as the SYSTEM user of windows. I also did not find a way to redirect the output to a file or directory.

When executing the steps with a logged in user everything works as expected, however, so I did not see the need to attach that output - the error simply does not exist there.
